CMakeLists.txt

```diff
@@ -17,68 +14,22 @@ endif()
 
 add_subdirectory(third_party/googletest)
 
-include_directories(SYSTEM ${gtest_SOURCE_DIR} ${gtest_SOURCE_DIR}/include)
-include_directories(SYSTEM "${PROJECT_SOURCE_DIR}/third_party/eigen/")
-include_directories(SYSTEM "${PROJECT_SOURCE_DIR}/third_party/cereal/include")
-include_directories(SYSTEM "${PROJECT_SOURCE_DIR}/third_party/fast-cpp-csv-parser/")
+add_library(albatross INTERFACE)
+target_include_directories(albatross INTERFACE
+  "${PROJECT_SOURCE_DIR}/include"
+  )
+target_include_directories(albatross SYSTEM INTERFACE
+  "${PROJECT_SOURCE_DIR}/third_party/eigen/"
+  "${PROJECT_SOURCE_DIR}/third_party/cereal/include"
+  "${PROJECT_SOURCE_DIR}/third_party/fast-cpp-csv-parser/"
+  )
+
 
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -Werror -fexceptions -frtti")
```

Comment:
want to fix that too?  take a look at what I did in orion...

Reply:
done

Reply:
just not pushed...?